### PR TITLE
MobaXterm: Fix path of mobaxterm.exe

### PR DESCRIPTION
**Pull request by cwbhhjl**
$ex is an absolute path of mobaxterm*.exe, so ‘Rename-Item’ will throw an error.

```
Rename-Item : Cannot find path 'D:\local\scoop\apps\mobaxterm\12.1\D:\local\scoop\apps\mobaxterm\12.1\MobaXterm_Personal_12.1.exe' because it does not exist."
```

### CODE REVIEW

File: bucket/mobaxterm.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "pre_install": [
         "    # Rename executable",
         "$ex = (Get-ChildItem \"$dir\" 'mobaxterm*.exe')[0]",
```
Comment (Ash258) [inline bucket/mobaxterm.json]:
```suggestion
        "$ex = Get-ChildItem \"$dir\" 'mobaxterm*.exe' | Select-Object -First 1 | Rename-Item -NewName 'MobaXterm.exe'",
```

File: bucket/mobaxterm.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
...
-        "Rename-Item \"$dir\\$ex\" \"$dir\\MobaXterm.exe\"",
+        "Rename-Item \"$ex\" \"$dir\\MobaXterm.exe\"",
```
Comment (Ash258) [inline bucket/mobaxterm.json]:
```suggestion
```

File: bucket/mobaxterm.json (hunk 1 above)
Comment (linsui) [reply bucket/mobaxterm.json]: Why keep $ex ?

Comment (Ash258) [reply bucket/mobaxterm.json]: Yeah it could be removed.

File: bucket/mobaxterm.json [hunk 3]
```diff
@@ -10,8 +10,7 @@
     "hash": "489e882ad5ddd7aaa2195d8102ea2bad7e374d8b209260a6d12ee3703229dc8d",
     "pre_install": [
         "    # Rename executable",
-        "$ex = (Get-ChildItem \"$dir\" 'mobaxterm*.exe')[0]",
-        "Rename-Item \"$ex\" \"$dir\\MobaXterm.exe\"",
+        "$ex = Get-ChildItem \"$dir\" 'mobaxterm*.exe' | Select-Object -First 1 | Rename-Item -NewName 'MobaXterm.exe'",
```
Comment (linsui) [inline bucket/mobaxterm.json]:
```suggestion
        "Get-ChildItem \"$dir\" 'mobaxterm*.exe' | Select-Object -First 1 | Rename-Item -NewName 'MobaXterm.exe'",
```

**Pull request merged**